dali/pipeline/operators/reader/reader_op_test.cc

```diff
@@ -119,4 +119,29 @@ TYPED_TEST(ReaderTest, SimpleTest) {
   return;
 }
 
+
+TYPED_TEST(ReaderTest, SequenceTest) {
+  Pipeline pipe(128, 1, 0);
+
+  pipe.AddOperator(
+      OpSpec("SequenceReader")
+      .AddArg("file_root", "/mnt/nvvl_data/data/540p/frames/train/")
```

Comment:
This is only for temporary validation. We will have to create some other test data, because we cannot add this to our repo. So this commit should be left out.